apps/i18n/common/en_us.json

```diff
@@ -394,12 +394,8 @@
   "delete": "Delete",
   "deleteAccount": "Delete Account",
   "deleteAccount_studentWarning": "Deleting your account will permanently erase all personal information, coursework, and projects connected to this account.",
-  "deleteAccount_teacherWarning1": "Deleting your account will permanently erase all personal information, coursework, projects, and professional learning information connected to this account after 28 days. It will also ",
-  "deleteAccount_teacherWarning2": "delete your sections and your students’ accounts ",
-  "deleteAccount_teacherWarning3": "that don’t have a personal login or aren’t in another teacher’s section. Please make sure you have the authority to delete these students’ education records before deleting your own account. ",
-  "deleteAccount_teacherWarning4": "Give these students a chance to keep using their Code.org accounts by ",
-  "deleteAccount_teacherWarning5": "sending home instructions for creating personal logins. ",
-  "deleteAccount_teacherWarning6": "Give them at least a few days to follow these instructions BEFORE you delete your account.",
+  "deleteAccount_teacherWarning1": "Deleting your account will permanently erase all personal information, coursework, projects, and professional learning information connected to this account after 28 days. **It will also delete your sections and your students’ accounts** that don’t have a personal login or aren’t in another teacher’s section. Please make sure you have the authority to delete these students’ education records before deleting your own account. ",
+  "deleteAccount_teacherWarning2": "Give these students a chance to keep using their Code.org accounts by [sending home instructions for creating personal logins.]({explanationUrl}) Give them at least a few days to follow these instructions BEFORE you delete your account.",
```

Comment:
we'll need to create these as new strings, so the old translations aren't carried over

Reply:
We should also take this opportunity to give these strings more descriptive names